New version of CARMAMsg is not compatible with the current CARMAPlatform

### Descriptive summary
CircleCI fails on the latest develop branch.
```
/opt/carma/src/CARMAPlatform/carmajava/route/src/main/java/gov/dot/fhwa/saxton/carma/route/RouteWaypoint.java:92: error: cannot find symbol
      newManeuver.setLength(maneuver.getLength());
                                    ^
  symbol:   method getLength()
  location: variable maneuver of type Maneuver
/opt/carma/src/CARMAPlatform/carmajava/route/src/main/java/gov/dot/fhwa/saxton/carma/route/RouteWaypoint.java:93: error: cannot find symbol
      newManeuver.setPerformers(maneuver.getPerformers());
                                        ^
  symbol:   method getPerformers()
  location: variable maneuver of type Maneuver
/opt/carma/src/CARMAPlatform/carmajava/route/src/main/java/gov/dot/fhwa/saxton/carma/route/RouteWaypoint.java:94: error: cannot find symbol
      newManeuver.setStartRoadwayLaneId(maneuver.getStartRoadwayLaneId());
                                                ^
  symbol:   method getStartRoadwayLaneId()
  location: variable maneuver of type Maneuver
/opt/carma/src/CARMAPlatform/carmajava/route/src/main/java/gov/dot/fhwa/saxton/carma/route/RouteWaypoint.java:95: error: cannot find symbol
      newManeuver.setStartRoadwayLink(maneuver.getStartRoadwayLink());
                                              ^
  symbol:   method getStartRoadwayLink()
  location: variable maneuver of type Maneuver
/opt/carma/src/CARMAPlatform/carmajava/route/src/main/java/gov/dot/fhwa/saxton/carma/route/RouteWaypoint.java:96: error: cannot find symbol
      newManeuver.setStartRoadwayOriginatorPosition(maneuver.getStartRoadwayOriginatorPosition());
                                                            ^
  symbol:   method getStartRoadwayOriginatorPosition()
  location: variable maneuver of type Maneuver
5 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':route:compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
CARMAPlatform/carmajava/CMakeFiles/gradle-carma.dir/build.make:57: recipe for target 'CARMAPlatform/carmajava/CMakeFiles/gradle-carma' failed
make[2]: *** [CARMAPlatform/carmajava/CMakeFiles/gradle-carma] Error 1
CMakeFiles/Makefile2:12254: recipe for target 'CARMAPlatform/carmajava/CMakeFiles/gradle-carma.dir/all' failed
make[1]: *** [CARMAPlatform/carmajava/CMakeFiles/gradle-carma.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2
Invoking "make install -j36 -l36" failed
Exited with code 1
```
But the code can be build locally with an old version of CARMAMsg package.
### Expected behavior
CircleCI should pass always.
### Actual behavior
CircleCI fails.
### Steps to reproduce the behavior
Build CARMAPlatform with the latest CARMAMsg package.
### Related work
https://github.com/usdot-fhwa-stol/CARMAMsgs/pull/7 breaks the dependency between them.

--- a/carmajava/route/src/main/java/gov/dot/fhwa/saxton/carma/route/RouteWaypoint.java
+++ b/carmajava/route/src/main/java/gov/dot/fhwa/saxton/carma/route/RouteWaypoint.java
@@ -86,16 +86,6 @@ public class RouteWaypoint {
     // String and Integer are immutable so add basic copy will work
     disabledGuidanceAlgorithms.addAll(wp.disabledGuidanceAlgorithms);
     laneClosures.addAll(wp.laneClosures);
-    // Deep copy needed maneuvers
-    for (cav_msgs.Maneuver maneuver: wp.neededManeuvers) {
-      cav_msgs.Maneuver newManeuver = messageFactory.newFromType(Maneuver._TYPE);
-      newManeuver.setType(maneuver.getType());
-      newManeuver.setLaneChangeManeuver(maneuver.getLaneChangeManeuver());
-      newManeuver.setIntersectionTransitStraightManeuver(maneuver.getIntersectionTransitStraightManeuver());
-      newManeuver.setIntersectionTransitLeftTurnManeuver(maneuver.getIntersectionTransitLeftTurnManeuver());
-      newManeuver.setIntersectionTransitRightTurnManeuver(maneuver.getIntersectionTransitRightTurnManeuver());
-      newManeuver.setStopAndWaitManeuver(maneuver.getStopAndWaitManeuver());
-    }
     // Copy remaining fields
     laneCount = wp.laneCount;
     this.setLocation(wp.getLocation());

